mavros/src/lib/uas_sensor_orientation.cpp

```diff
@@ -14,63 +14,64 @@
  * https://github.com/mavlink/mavros/tree/master/LICENSE.md
  */
 #include <array>
+#include <angles/angles.h>
 #include <mavros/mavros_uas.h>
 
 using namespace mavros;
 
-// XXX do we need combine that with string representation?
+/** @todo Combine the bellow with string representation */
 
-static const std::array<const tf::Vector3, 39> sensor_orientation = {
-	/*  0 */ tf::Vector3(0.0,   0.0,    0.0),
-	/*  1 */ tf::Vector3(0.0,   0.0,    45.0),
-	/*  2 */ tf::Vector3(0.0,   0.0,    90.0),
-	/*  3 */ tf::Vector3(0.0,   0.0,    135.0),
-	/*  4 */ tf::Vector3(0.0,   0.0,    180.0),
-	/*  5 */ tf::Vector3(0.0,   0.0,    225.0),
-	/*  6 */ tf::Vector3(0.0,   0.0,    270.0),
-	/*  7 */ tf::Vector3(0.0,   0.0,    315.0),
-	/*  8 */ tf::Vector3(180.0, 0.0,    0.0),
-	/*  9 */ tf::Vector3(180.0, 0.0,    45.0),
-	/* 10 */ tf::Vector3(180.0, 0.0,    90.0),
-	/* 11 */ tf::Vector3(180.0, 0.0,    135.0),
-	/* 12 */ tf::Vector3(0.0,   180.0,  0.0),
-	/* 13 */ tf::Vector3(180.0, 0.0,    225.0),
-	/* 14 */ tf::Vector3(180.0, 0.0,    270.0),
-	/* 15 */ tf::Vector3(180.0, 0.0,    315.0),
-	/* 16 */ tf::Vector3(90.0,  0.0,    0.0),
-	/* 17 */ tf::Vector3(90.0,  0.0,    45.0),
-	/* 18 */ tf::Vector3(90.0,  0.0,    90.0),
-	/* 19 */ tf::Vector3(90.0,  0.0,    135.0),
-	/* 20 */ tf::Vector3(270.0, 0.0,    0.0),
-	/* 21 */ tf::Vector3(270.0, 0.0,    45.0),
-	/* 22 */ tf::Vector3(270.0, 0.0,    90.0),
-	/* 23 */ tf::Vector3(270.0, 0.0,    135.0),
-	/* 24 */ tf::Vector3(0.0,   90.0,   0.0),
-	/* 25 */ tf::Vector3(0.0,   270.0,  0.0),
-	/* 26 */ tf::Vector3(0.0,   180.0,  90.0),
-	/* 27 */ tf::Vector3(0.0,   180.0,  270.0),
-	/* 28 */ tf::Vector3(90.0,  90.0,   0.0),
-	/* 29 */ tf::Vector3(180.0, 90.0,   0.0),
-	/* 30 */ tf::Vector3(270.0, 90.0,   0.0),
-	/* 31 */ tf::Vector3(90.0,  180.0,  0.0),
-	/* 32 */ tf::Vector3(270.0, 180.0,  0.0),
-	/* 33 */ tf::Vector3(90.0,  270.0,  0.0),
-	/* 34 */ tf::Vector3(180.0, 270.0,  0.0),
-	/* 35 */ tf::Vector3(270.0, 270.0,  0.0),
-	/* 36 */ tf::Vector3(90.0,  180.0,  90.0),
-	/* 37 */ tf::Vector3(90.0,  0.0,    270.0),
-	/* 38 */ tf::Vector3(315.0, 315.0,  315.0)
+static const std::array<const Eigen::Vector3d, 39> sensor_orientation = {
```

Comment:
Really that used once and Eigen::Quaterniond is more usable form of rotation.
So i want redo this at all.
My steps:
1. make static initializer function, maybe **constexpr** (if eigen can handle it)
 this function will be used only in that module, so it should be static.
2. replace all calls to that func